data-pipeline/src/pipeline/pre_processing.py

```diff
@@ -997,23 +1047,13 @@ def build_academy_data(
         - academies["Income_Catering services"]
     )
 
-    academies["Catering staff and supplies_Net Costs_CS"] = (
-        academies["Catering staff and supplies_Total_CS"]
-        - academies["Income_Catering services_CS"].fillna(0.0)
-    )
-
-    trust_revenue_reserve = (
-        academies[["Trust UPIN", "Revenue reserve"]]
-        .groupby(["Trust UPIN"])
-        .sum()
-        .rename(
-            columns={
-                "Revenue reserve": "Trust Revenue reserve",
-            }
-        )
-    )
+    academies["Catering staff and supplies_Net Costs_CS"] = academies[
+        "Catering staff and supplies_Total_CS"
+    ] - academies["Income_Catering services_CS"].fillna(0.0)
 
-    academies = academies.merge(trust_revenue_reserve, on="Trust UPIN", how="left")
+    academies = _trust_revenue_reserve(academies, central_services)
+    # TODO: `Revenue reserve_CS` to be removed.
+    academies["Revenue reserve_CS"] = 0.0
```

Comment:
The ends up as `RevenueReserveCS` in both `Financial` and TrustFinancial` so although it's defunct, it needs some checking to make sure it can be removed.

Reply:
Discussion scheduled to update as required and remove this.